cmd/backup/config.go

```diff
@@ -58,6 +61,14 @@ type Config struct {
 	LockTimeout                time.Duration `split_words:"true" default:"60m"`
 }
 
+func (c *Config) ResolveSecret(envVar string, secretPath string) (string, error) {
```

Comment:
There's no need to make this method public I think, it can be called `resolveSecret` instead.